README.md

```diff
@@ -16,12 +16,79 @@
 
 ## About
 
-:construction: WIP
+Benchttp engine is a Go library providing a way to perform benchmarks and tests
```

Comment:
Nitpick: it's not _really_ a lib in the way that it only has one way to be used. I think package don't bring that weight of composability with it.
```suggestion
Benchttp engine is a Go library package a way to perform benchmarks and tests
```

Reply:
I agree, package sound a little more suitable for me too.
```suggestion
Benchttp engine is a Go package providing a way to perform benchmarks and tests
```

Reply:
To me it's still a library, as opposed to an executable application — I felt important to make that clear, as "package" is used for both world in Go.

Reply:
It makes sense too ! I leave judge on that :)

Reply:
I have no strong argument about it either, so OK for library if you prefer it.